app/src/ui/publish-repository/publish-repository.tsx

```diff
@@ -109,7 +110,10 @@ export class PublishRepository extends React.Component<
 
     let selectedIndex = -1
     const selectedOrg = this.props.settings.org
-    for (const [index, org] of this.state.orgs.entries()) {
+    const orgs = [...this.state.orgs.entries()].sort(([, a], [, b]) =>
```

Comment:
Also, this doesn't quite seem to be sorting as I'd expected:

<img width="160"  src="https://user-images.githubusercontent.com/359239/34374532-a66799c6-eb35-11e7-9a45-d2c36fac1bb3.png">

This is just a snippet of my orgs, I can't quite tell what the pattern is that it is sorting on...